package/functions/parser.js

```diff
@@ -310,7 +306,7 @@ const Parser = {
   $rolePerms:
     "Returns the role key permissions.;$rolePerms[roleID;separator (optional)]",
   $songInfo:
-    "Displays information about the song that is being played.;$songInfo[property;position]",
+    "Displays information about the song that is being played.;$songInfo[property]",
```

Comment:
Might want to put newer parser.js, then add the function

Reply:
mk, i'll change that too